chemclipse/tests/org.eclipse.chemclipse.msd.model.fragment.test/src/org/eclipse/chemclipse/msd/model/xic/ExtractedIonSignalsModifier_2_Test.java

```diff
@@ -116,67 +100,52 @@ public void testAdjustThresholdTransitions_1() {
 			 * Test the adjusted value.
 			 */
 			extractedIonSignal = extractedIonSignals.getExtractedIonSignal(4);
-			assertEquals("TotalIonSignal after", 35.054245f, extractedIonSignal.getAbundance(45));
-			assertEquals("TotalIonSignal after", 35.054245f, extractedIonSignal.getAbundance(76));
-			assertEquals("TotalIonSignal after", 35.054245f, extractedIonSignal.getAbundance(102));
+			assertEquals("TotalIonSignal after", 35.054245f, extractedIonSignal.getAbundance(45), 0);
+			assertEquals("TotalIonSignal after", 35.054245f, extractedIonSignal.getAbundance(76), 0);
+			assertEquals("TotalIonSignal after", 35.054245f, extractedIonSignal.getAbundance(102), 0);
 		} catch(AnalysisSupportException e) {
```

Comment:
Remove the catch clauses and let the method throw the exceptions for better stacktrace in case of failure.